env0/data_aws_credentials_test.go

```diff
@@ -0,0 +1,63 @@
+package env0
+
+import (
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAwsCredDataSource(t *testing.T) {
+	awsCred := client.ApiKey{
+		Id:   "11111",
+		Name: "testdata",
+	}
```

Comment:
you are missing `Type` property here

since we filter by `Type` 

https://github.com/env0/terraform-provider-env0/blob/80f1837e0bf64ef77c620a58d7f38fe6850e680b/env0/data_aws_credentials.go#L85-L87

and 

https://github.com/env0/terraform-provider-env0/blob/80f1837e0bf64ef77c620a58d7f38fe6850e680b/env0/data_aws_credentials.go#L67-L71